environment.yml

```diff
@@ -44,3 +44,5 @@ dependencies:
  - tqdm
  - dem_stitcher>=2.4.0
  - aiohttp  # only needed for manifest and swath download
+ - pip:
+   - tile-mate
```

Comment:
this is on conda-forge now, isn't it?

Reply:
Yes - but it was slightly delayed and I was impatient - I will change before I merge.

Reply:
Had to expand admissible python versions of tile-mate to `3.9` (was previously `>=3.10`). Getting better at the ASF release process! Thank you, Joe!